src/test/java/com/hocs/test/pages/decs/Search.java

```diff
@@ -962,7 +962,7 @@ public boolean zeroSearchResultsReturned() {
         return getNumberOfSearchResults() == 0;
     }
 
-    public void selectCOMP2CaseRefOfEscalatedCOMPCase(String stage1CaseRef) {
+    public void selectComplaintsStage2CaseRefOfEscalatedCOMPCase(String stage1CaseRef) {
         WebElementFacade comp2CaseRef = findBy("//a[text()='" + stage1CaseRef + "']/parent::td/following-sibling::td/a");
```

Comment:
This variable and half of the method name still are for COMP, did we need to label this for use in all complaints escalations?